README.md

```diff
@@ -10,8 +10,9 @@ Front-end for TIS Self-Service app
 
 Full Tech and Architecture are described elsewhere. The front-end uses REST interaction with the back-end service(s).
 
-[Create React App (CRA)](https://create-react-app.dev/) is used to build the app as it comes with many build tools already configured.
-This single page application (SPA) is written in [TypeScript](https://www.typescriptlang.org/) using the [React](https://reactjs.org/) library.
+[Next.js](https://nextjs.org/) is used to build the app. It is now the industry standard for building an enterprise-level React applications. It gives lots of out-the-box features such as code-splitting and build optimization and allows full control over the underlying configuration.It will also allow for SSR as and when we decide (which can be on a component-by-component basis) which will improve UX.
```

Comment:
Missing whitespace 
> configuration.It

Reply:
@Judge40 Are you any good at Where's Wally? 👓